staging/src/k8s.io/kube-aggregator/pkg/apiserver/metrics.go

```diff
@@ -46,7 +50,36 @@ var x509InsecureSHA1Counter = metrics.NewCounter(
 	},
 )
 
+var (
+	extApiserverAttemptsCounter = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "extension_apiserver_attempts_total",
+			Help:           "Counter of extension apiserver attempts broken down by result. It can be either 'OK', 'Not Found', 'Service Unavailable' or 'Internal Server Error'.",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"result"},
```

Comment:
Just like webhooks, we should also add a dimension here on the name of the extension apiserver imo.

@liggitt @deads2k  - looking for your inputs from apimachinery side :)

Reply:
I wouldn't call it `attempts`, I prefer `requests`

Reply:
I meant `extension_apiserver_requests_total`.

Reply:
+1. The lack of apiservice name or something like that surprises.